commands/version/__tests__/version-github-release.test.js

```diff
@@ -19,19 +19,19 @@ const initFixture = require("@lerna-test/init-fixture")(__dirname);
 // test command
 const lernaVersion = require("@lerna-test/command-runner")(require("../command"));
 
-test("--github-release does not create a release if --no-push is passed", async () => {
+test("--release does not create a release if --no-push is passed", async () => {
```

Comment:
Might as well revert this file, as the tests _should_ still work if you've configured the hidden option fallback correctly.